contracts/interfaces/IERC5313.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/// @title EIP-5313 Light Contract Ownership Standard
+interface IERC5313 {
+    /// @notice Get the address of the owner
+    /// @return The address of the owner
+    function owner() external view returns (address);
+}
```

Comment:
This will conflict with #4009. I can merge back but I'd still advocate for docs consistent with ours

```suggestion
/**
 * @dev Interface for the Light Contract Ownership Standard.
 *
 * A standardized minimal interface required to identify an account that controls a contract
 *
 * _Available since v4.9._
 */
interface IERC5313 {
    /**
     * @dev Gets the address of the owner.
     */
    function owner() external view returns (address);
}
```

Both `@notice` and `@title` are not picked up from our docs generation

Reply:
I just copied the interface provided in the (final) EIP. How is that in conflict with #4009? If that is the case, #4009 should either not advertise 5313 compatibility ... of change to be compatible

Reply:
Conflicts because I added it as well there to make it 5313 compatible with different comments, just that.